addons/point_of_sale/static/src/js/Popups/MoneyDetailsPopup.js

```diff
@@ -36,7 +36,7 @@ export class MoneyDetailsPopup extends AbstractAwaitablePopup {
             (total, money) => total + money[0] * money[1],
             0
         );
-        this.state.total = this.pos.globalState.round_decimals_currency(total);
```

Comment:
Maybe we can get around this by just targetting directly the "view" that displays this value.

Reply:
It turned out that `state.total` is formatted in the template.